builtins/zeromq/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+include(ExternalProject)
+
+set(ZeroMQ_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/BUILTIN_ZeroMQ-prefix)
+set(ZeroMQ_LIBNAME ${CMAKE_STATIC_LIBRARY_PREFIX}zmq${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+set(ZeroMQ_LIBRARY ${ZeroMQ_PREFIX}/lib/${ZeroMQ_LIBNAME} CACHE INTERNAL "" FORCE)
+set(ZeroMQ_LIBRARIES ${ZeroMQ_LIBRARY} CACHE INTERNAL "" FORCE)
+
+ExternalProject_Add(BUILTIN_ZeroMQ
+    URL https://github.com/zeromq/libzmq/archive/refs/tags/v4.3.2.tar.gz
+    CMAKE_ARGS
+        -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+        -DWITH_PERF_TOOL=OFF
+        -DZMQ_BUILD_TESTS=OFF
+        -DPOLLER=select
+        # TODO: remove -w flag when updating to v4.3.5 or higher, current warnings should be gone by then
+        -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}\ -w
```

Comment:
```cmake
-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
```
Please also use the same compiler as for the rest of ROOT. Otherwise, the compilation can fail because of incompatibilities.

Reply:
Done in c95ee93.